packages/api-client/src/views/Request/RequestSection/RequestAuth/RequestAuth.vue

```diff
@@ -288,6 +289,24 @@ const openAuthCombobox = (event: Event) => {
               weight="bold"
               class="size-3 shrink-0 transition-transform duration-100 group-aria-expanded/combobox-button:rotate-180" />
           </ScalarButton>
+          <template #option="{ option, selected }">
+            <ScalarListboxCheckbox
+              :selected="selected"
+              multiselect />
+            <div class="min-w-0 flex-1 truncate">
+              {{ option.label }}
+            </div>
+            <ScalarIconButton
```

Comment:
This used to be a clickable icon but technically a button is more semantically correct